Assets/HoloToolkit/Input/Scripts/Utilities/Managers/InputManager.cs

```diff
@@ -233,15 +373,15 @@ private void Start()
 
         #endregion // Unity APIs
 
-        public void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.EventFunction<T> eventHandler) where T : IEventSystemHandler
+        public void HandleEvent<T>(BaseInputEventData eventData, ExecuteEvents.EventFunction<T> eventHandler) where T : IEventSystemHandler
```

Comment:
What's the need/benefit to change this from the more generic `BaseEventData`?

Reply:
None (yet) - I'll revert this for now.